pkg/operator/ceph/file/mds/mds.go

```diff
@@ -127,6 +128,16 @@ func (c *Cluster) Start() error {
 	// Always create double the number of metadata servers to have standby mdses available
 	replicas := c.fs.Spec.MetadataServer.ActiveCount * 2
 
+	mdsToSkipReconcile, err := mon.ToSkipReconcile(c.clusterInfo.Context, c.context, c.clusterInfo.Namespace, config.MdsType, AppName)
+	if err != nil {
+		return errors.Wrap(err, "failed to check for mds to skip reconcile")
+	}
+
+	if mdsToSkipReconcile.Len() > 0 {
```

Comment:
This check needs to be inside the `for` loop below, so that only the deployments that are labeled will be skipped. If they want to skip all the mds deployments, they should label all of them.

Reply:
done